TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -219,6 +219,66 @@ const BenchmarkMetricRegex = {
                 funcName: geomean,
             }
         }
+    },
+    quarkus_startup: {
+        outerRegex: /Quarkus Test:  measure \d*? ([\s\S\n]*)/,
```

Comment:
Just as we've added examples for `regex`, let's do it for `outerRegex` as well for the newly added.

Reply:
I've updated our script for it by removing the unnecessary space.